dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESIndexAPI.java

```diff
@@ -613,7 +613,7 @@ public synchronized CreateIndexResponse createIndex(final String indexName, Stri
 		map.put("number_of_shards", shards);
 		map.put("index.auto_expand_replicas", "0-all");
 		if (!map.containsKey("index.mapping.total_fields.limit")) {
-			map.put("index.mapping.total_fields.limit", 10000);
+			map.put("index.mapping.total_fields.limit", 11000);
```

Comment:
Why do you need to increment this number? if it's really necessary I suggest to test with a large dataset to make sure it doesn't hit the performance

Reply:
I don't see new index fields on these changes. I'd really take a look

Reply:
done https://github.com/dotCMS/core/pull/19955/commits/2bba46387c6412fc00a7a021a2feae5e9d005d3c#diff-13f591fe344d5a9a147ae7bc54365b8fd937a8daf84bff07d423c2c1bc309034R616